.devcontainer/on-create-command.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+if ! git remote | grep -q "fork"; then
```

Comment:
Since the guide instructs contributors to fork first, is this necessary? When I open a codespace on this PR, I see your fork as `origin`, and pallets as `upstream`.

Additionally, is there any way to automate _creating_ the fork so that users can start the codespace from pallets? I think the basic GitHub editor does this.

Reply:
Good catch - I've removed this section and added instructions for automatic forking.